test-suite/hestonmodel.cpp

```diff
@@ -1940,6 +2684,19 @@ test_suite* HestonModelTest::suite(SpeedLevel speed) {
                     &HestonModelTest::testAllIntegrationMethods));
     suite->add(QUANTLIB_TEST_CASE(&HestonModelTest::testCosHestonCumulants));
     suite->add(QUANTLIB_TEST_CASE(&HestonModelTest::testCosHestonEngine));
+    suite->add(QUANTLIB_TEST_CASE(&HestonModelTest::testCharacteristicFct));
+    suite->add(QUANTLIB_TEST_CASE(
```

Comment:
These can go in the "faster" section, right?

Reply:
Sorry, I haven't fully understood your proposal. Isn't "faster" the default? Or do you want to move them to the fast/slow section.

Reply:
Right, I wasn't clear.  They are already in the faster section.  My question was: they're fast enough for it, right?

Reply:
The seven new tests are taking less than a second. I'd leave them in the "faster" section. Do you have a criteria to decide where to put new tests, e.g. everything below 1 second goes to faster?